corehq/apps/domain/templates/login_and_password/domain_mobile_reset_pwd_email.html

```diff
@@ -0,0 +1,16 @@
+{% load i18n %}{% autoescape off %}
+{% blocktrans %}You're receiving this email because you requested a password reset for your user account at {{ site_name }}{% endblocktrans %}.
+
+{% trans "Please go to the following page and choose a new password:" %}
+{% block reset_link %}
+{{ protocol }}://{{ domain }}{% url "domain_mobile_reset_pwd_confirm" domain=domain_name uidb64=uid token=token %}
+{% endblock %}
+{% trans "Your username, in case you've forgotten:" %} {{ user.username }}
```

Comment:
I agree. This template is a copy of the one sent to web users. I'll change the wording here, and fix the web users' template in a separate branch/PR.